docs/config/README.md

```diff
@@ -95,6 +95,7 @@ prompt_order = [
     "hostname",
     "kubernetes",
     "directory",
+    "elm",
```

Comment:
This should be moved down, together with the other languages/frameworks

Reply:
Fixed. 👍